wcfsetup/install/lang/de.xml

```diff
@@ -1645,6 +1645,7 @@ Wenn Sie unter <em>System -&gt; Optionen -&gt; Allgemein -&gt; E-Mails</em> alle
 		<item name="wcf.acp.user.option.editable.1"><![CDATA[Eigentümer]]></item>
 		<item name="wcf.acp.user.option.editable.2"><![CDATA[Administrator]]></item>
 		<item name="wcf.acp.user.option.editable.3"><![CDATA[Eigentümer und Administrator]]></item>
+		<item name="wcf.acp.user.option.editable.8"><![CDATA[Eigentümer bei Erstanmeldung und Administrator]]></item>
```

Comment:
This does not really make sense. Each bit is for a specific permission. You would use `4` for Eigentümer bei Erstanmeldung and `6` would be both, as `4` + `2` is `6`.